main.py

```diff
@@ -219,10 +218,15 @@ def _switch_colors(self) -> dict:
     def _send_flow_mods(self, dpid_flows: dict) -> None:
         """Send FlowMods."""
```

Comment:
Should probably wrap this loop into an httpx client, then make all the post requests through the client.

Reply:
Now using client.